### PR TITLE
Generate SSH keys within module

### PR DESCRIPTION
**Pull request by sgryczan**
Addresses https://github.com/NetApp/ez-rancher/issues/36.

No longer generating SSH key pair while building the deployment container. 

This PR moves `ssh-keygen` functionality into the infrastructure module, so that a new random key pair will be generated prior to creating the cluster nodes. Keys will continue to be exported to the deliverables directory, and their lifecycle will be tied to that of the TF project.

### CODE REVIEW

File: .github/workflows/workflow.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ on:
         description: Git Ref (Optional)    
         required: false
   schedule:
-    - cron: '45 * * * *'
+    - cron: '0 */4 * * *'
```
Comment (sgryczan) [inline .github/workflows/workflow.yaml:14]: Slightly reducing the frequency of scheduled builds

Comment (bgentry-ntap) [reply .github/workflows/workflow.yaml:14]: Can you add a comment explaining the frequency? Is this every 1/4 hour?

**Comment by j-griffith** [comment]
/LGTM
We can add an authroized_keys option later if we want

File: .github/workflows/workflow.yaml (hunk 1 above)
Comment (sgryczan) [reply .github/workflows/workflow.yaml:15]: Done

**Pull request merged**